README.md

```diff
@@ -1,18 +1,18 @@
 [![Malimbe logo][Malimbe-Image]](#)
 
 > ### Malimbe
-> A collection of tools to simplify writing public API components in Unity.
+> A collection of tools to simplify writing public API components for the Unity software.
 
 [![License][License-Badge]][License]
 [![Waffle][Waffle-Badge]][Waffle]
 
 ## Introduction
 
-Malimbe is a collection of tools to simplify writing public API components in Unity.
+Malimbe is a collection of tools to simplify writing public API components in the [Unity] software.
```

Comment:
I know it's kinda unrelated to this PR but I don't think we need another PR just for this: Let's delete this whole paragraph so we don't repeat ourselves like we do up in the `### Malimbe` header...

Reply:
Rather than just completely delete the line and have the Introduction start with `By` how about it just gets reworded to something like

`Malimbe for the [Unity] software aims to reduce repetitive boilerplate code by taking the assemblies that are created by build tools and changing the assembly itself, new functionality can be introduced and logic written as part of the source code can be altered. This process is called Intermediate Language (IL) weaving and Malimbe uses [Fody] to do it.`

(otherwise we start a paragraph with By which sounds weird, and there's no top level link to Unity and it looks weird when the link is in the note field)

Reply:
Awesome, that works out nicely! 👍